src/frontend/context/SimuleringContext.tsx

```diff
@@ -238,14 +233,9 @@ const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProp
     ): ISimuleringPeriode {
         if (
             periode.resultat === 0 &&
-            kalenderDiff(
-                kalenderDatoTilDate(
-                    periode.forfallsdato ? kalenderDato(periode.forfallsdato) : TIDENES_MORGEN
-                ),
-                kalenderDatoTilDate(
-                    tidSimuleringHentet ? kalenderDato(tidSimuleringHentet) : TIDENES_MORGEN
-                )
-            ) > 0
+            periode.forfallsdato &&
+            tidSimuleringHentet &&
+            isAfter(parseIsoString(periode.forfallsdato), parseIsoString(tidSimuleringHentet))
```

Comment:
Hva skjer når `periode.forfallsdato` er undefined her?

Reply:
Da er uttrykket false. Hvis enten `periode.forfallsdato` eller `tidSimuleringHentet` er undefined vil aldri det gamle uttrykket bli true. For da vil man sjekke om TIDENES_MORGEN er etter tidSimulering, og det vil aldri være true. Men, ser nå at man har brukt TIDENES_MORGEN som default for tidSimuleringHentet (har tenkt at det sto TIDENES_ENDE), så jeg må endre litt her for at det skal bli likt som før.

Reply:
Men usikker på om det som har vært før er riktig? Skjønner egentlig ikke hvorfor det er frontend som rører borti den delen her. Burde ikke det som kommer fra backend være riktig? 🤔

Reply:
Tror jeg trenger litt hjelp for å skjønne denne og hva som blir riktig oppførsel 😅

Reply:
Det er lagt til i denne committen: https://github.com/navikt/familie-ba-sak-frontend/commit/10da9be4672ac1bb357d9579c41ec75d8a508138

Reply:
Ser at det er du som har godkjent den @halvorbmundal, kanskje du husker 😆

Reply:
Kan alternativt gjøre sånn her så skal oppførselen være helt lik som i dag:
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/0375285f-e4c8-4eab-8cd3-871516627249)

Reply:
Men syns det høres litt rart ut at man bruker tidenes morgen som fallback for tidSimuleringHentet. Fordi det betyr at hvis tidSimuleringHentet er undefined, så vil denne alltid være true og da setter man alltid perioden til å være ikke utbetalt?